qt/lib/qtscenegraphdevicecontext.h

```diff
@@ -0,0 +1,190 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        qtscenegraphdevicecontext.h
+// Author:      Jonathan Schluessler
+// Created:     2017
+// Copyright (c) Authors and others. All rights reserved.
+/////////////////////////////////////////////////////////////////////////////
+
+#ifndef __QTSCENEGRAPHDEVICECONTEXT_H__
+#define __QTSCENEGRAPHDEVICECONTEXT_H__
+
+//----------------------------------------------------------------------------
+
```

Comment:
I had to include `<cmath>` for this file to compile

Reply:
I added <cmath> to the .cpp file which uses std::sin etc. I couldn't find anything in the .h file that might require <cmath>.